lib/charms/traefik_k8s/v0/ingress_per_unit.py

```diff
@@ -116,18 +116,26 @@ def _handle_ingress_per_unit(self, event):
 }
 
 
+# ======================= #
+#          TYPES          #
+# ======================= #
+
+
+KeyValueMapping = Dict[str, str]
+RequirerUnitData = Dict[Unit, KeyValueMapping]
+ProviderApplicationData = Dict[str, KeyValueMapping]
+
+
 # ======================= #
 #  SERIALIZATION UTILS    #
 # ======================= #
```

Comment:
```suggestion

KeyValueMapping = Dict[str, str]
RequirerUnitData = Dict[Unit, KeyValueMapping]
ProviderApplicationData = Dict[str, KeyValueMapping]

```

I would omit these comments, or at least tone them down.

Reply:
Agree, I'm not a big fan.

Reply:
done!